app/kumactl/cmd/install/context/install_control_plane_context.go

```diff
@@ -123,6 +124,7 @@ func DefaultInstallCpContext() InstallCpContext {
 			Egress_enabled:                          false,
 			Egress_drainTime:                        "30s",
 			Egress_service_type:                     "ClusterIP",
+			Ebpf_bpffspath:                          "/run/kuma/bpf",
```

Comment:
We agreed on not adding more params here and simply rely on regular values from helm instead.